awx/ui/client/src/templates/job_templates/multi-credential/multi-credential-modal.directive.js

```diff
@@ -111,7 +111,7 @@ function multiCredentialModalController(GetBasePath, qs, MultiCredentialService)
 
         scope.credentialTypes.forEach((credentialType => {
             if(credentialType.kind
-                .match(/^(machine|cloud|net|ssh|vault)$/)) {
+                .match(/^(machine|cloud|net|ssh|vault|kubernetes)$/)) {
                     scope.displayedCredentialTypes.push(credentialType);
             }
         }));
```

Comment:
![Screenshot from 2020-07-16 09-16-03](https://user-images.githubusercontent.com/9753817/87675426-5e7ad300-c745-11ea-82f9-1cccb9c5668e.png)

Icon missing on form, you can add it here: https://github.com/ansible/awx/blob/devel/awx/ui/client/src/templates/job_templates/multi-credential/multi-credential.partial.html#L22-L37